package.json

```diff
@@ -16,7 +18,7 @@
     "eslint-plugin-standard": "^4.0.1"
   },
   "dependencies": {
-    "mux-embed": "^3.2"
```

Comment:
I think we want to make this `^3.4`

The way semver works is:

* `^3.4` will install anything greater than or equal to `3.4` but less than `4`

Putting `^3.4.2` like you have means it will do greater than `3.4.2` but less than `3.4.5`. When we release mux-embed `3.5` we do not introduce breaking changes so we will want users of this react native SDK to pull in 3.5, 3.6, 3.7, etc.

Semver can be confusing....